test/unit/pdf_application_form_test.rb

```diff
@@ -0,0 +1,38 @@
+require 'test_helper'
+
+class PdfApplicationFormTest < ActiveSupport::TestCase
+  PDF_FIXTURE = './test/fixtures/pdfs/form_1.pdf'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.